azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -356,23 +345,22 @@ def execute_streaming_ingest(
             request_params["mappingName"] = mapping_name
 
         request_headers = {
-            "Accept-Encoding": accept_encoding,
-            "Connection": connection,
+            "Accept": "application/json",
+            "Accept-Encoding": "gzip,deflate",
             "x-ms-client-version": "Kusto.Python.StreamingClient:" + VERSION,

```

Comment:
That is not a streaming client. I'd redommend: 
"x-ms-client-version": "KPC.StreamingIngest." + VERSION,